src/assets/stylesheets/base_styles/_typography.scss

```diff
@@ -1,9 +1,9 @@
 @import 'colors';
 
-$font-family-serif:           'PolicyGenius', Georgia, serif;
-$font-family-sans-serif:      'PolicyGenius', Arial, sans-serif;
-$font-family-sans-serif-bold: 'PolicyGenius', Arial, sans-serif;
-$font-family-c: Georgia;
+@import url('https://fonts.googleapis.com/css2?family=Lora:wght@400;700&display=swap');
```

Comment:
The `'PolicyGenius'` font family is fetched from [this link](https://static.policygenius.com/fonts/PolicyGenius/font.css), and is just an alias for the `SharpSans` font. I didn't see a need for this layer of indirection for the new Lora font, especially since it is freely available and not a paid font.

Reply:
Shouldn't we keep the font in that bucket just in case google decides to mess with the font file without telling anyone though?

Reply:
@maxnovak I don't think we're deleting the older font. We wrapped the SharpSans font in Policygenius font, which added more complexity for not a lot of gain. 

```
Policygenius ->
 font weight: 400  -> SharpSans
 font weight: 900 -> SharpSans Display No 1 
```

Reply:
As for your point though @maxnovak, it's valid though I don't think Google will just take down fonts once they've been made accessible. We could inter it into our own CDN to be extra extra safe, but I don't really think that's super necessary. And even if it did fail to load, we have a fallback (and even then, this font doesn't look _that_ different from the default Times New Roman anyway imo 😂 )

Reply:
Ack, I misunderstood. Lora in our bucket, 🤦  That said, Agree with Ken.